Detectors/MUON/MID/Workflow/src/RawWriterSpec.cxx

```diff
@@ -14,7 +14,7 @@
 /// \date   02 October 2019
 
 #include "MIDWorkflow/RawWriterSpec.h"
-
+#include <TSystem.h>
```

Comment:
Can't we use the std::filesystem here instead of the ROOT TSystem?

Reply:
Unfortunately, no: the filesystem is not portable, e.g on ubuntu it is ``experimental/filesystem`` etc.

Reply:
Hi Ruben,
ok I see. Thanks for the explanation!